robottelo/api/utils.py

```diff
@@ -891,3 +891,27 @@ def attach_custom_product_subscription(prod_name=None, host_name=None):
         query={'search': 'name={0}'.format(prod_name)})[0]
     entities.HostSubscription(host=host.id).add_subscriptions(
         data={'subscriptions': [{'id': product_subscription.id, 'quantity': 1}]})
+
+
+def update_provisioning_template(name=None, old=None, new=None, org_id=None):
+    """ Update provisioning template content
+
+    :param str name: template provisioning name
+    :param str old: current content
+    :param str new: replace content
+    :param int org_id: organization ID
+
+    :return boolean: True/False
+    """
+    temp = entities.ProvisioningTemplate().search(
+        query={
+            'per_page': 1000,
+            'search': 'name="{}"'.format(name),
+            'organization_id': org_id
```

Comment:
I dont think that org id is required. OR we shouldn't make it mandatory.

Reply:
It was added to restrict the search with specific org only because in provisioning scenario we create a new org and templates are aligned to that only .

If you think it could make some false call .. I am happy to remove it .

Reply:
Provisioning Templates are global and they are not bound to any perticular org.

Hence, change in provisinoing template is univesal and not bound to any perticular taxonomy. If you change in template from org, the other org will show same change.

Hence, it doesnt make sense for org parameter here.

To reinitialize things, resetting the 'old' text in teardown is mandatory.

Reply:
True Provisioning Templates are global but they only become part of an organization if the get assigned to that org else they never get visible to it .
i.e.
![snap123](https://user-images.githubusercontent.com/23010485/63001756-f0bd8b80-be91-11e9-961b-368d51e72d27.png)

in above the `Kickstart default` template never be available to org `test123` .
hence a org requires for a provisioning template to become visible into that.

Now coming to the point: by default newly created org does have the all templates assigned with it.
I understand removing it  does not impact anything in functionality thus removing it now ..

Reply:
Removed in latest snap .. I hope good to merge this PR now :)